spec/controllers/ems_common_controller_spec.rb

```diff
@@ -130,5 +130,36 @@
         ManageIQ::Providers::Kubernetes::ContainerManager.last.authentication_token("bearer").should == "valid-token"
       end
     end
+
+    context "#button" do
+      before(:each) do
+        set_user_privileges
+        FactoryGirl.create(:vmdb_database)
+        EvmSpecHelper.create_guid_miq_server_zone
+      end
+
+      it "when VM Migrate is pressed for unsupported type" do
+        controller.stub(:role_allows).and_return(true)
+        controller.stub(:performed?).and_return(true)
```

Comment:
@h-kataria : should we stub this? I'd say that we are testing that we have the render logic ok and `performed?` is part of the logic. But it my be just that I don't get something here ;-)